kernel/src/main/java/org/apache/shardingsphere/elasticjob/kernel/internal/server/ServerService.java

```diff
@@ -132,7 +132,7 @@ public int removeOfflineServers() {
                 return;
             }
             String status = jobNodeStorage.getJobNodeData(serverNode.getServerNode(serverIp));
-            if (StringUtils.isNotBlank(status)) {
+            if (StringUtils.isBlank(status)) {
```

Comment:
- Is this actually corresponding to `org.apache.shardingsphere.elasticjob.kernel.internal.server.ServerStatus`?
```java
public enum ServerStatus {
    ENABLED, DISABLED
}
```
- Then why not just `return` when `ENABLED` is determined?

Reply:
- Only offline servers return null, online servers can return enabled or disabled
- We should not remove the server while it is still online, even if the job is disabled.
@linghengqian